.github/workflows/se_req_mve_pr_workflow.yml

```diff
@@ -26,10 +26,27 @@ on:
 # Execute platform_pr_dev_workflow workflow for experiment, evaluation and deployment of flows
 #=====================================
 jobs:
+  create-rule-id-list:
+      name: prompt flow experiment and evaluation job
+      runs-on: ubuntu-latest
+      outputs:
+        RULE_IDS: ${{ steps.rule_ids.outputs.RULE_IDS }}
+      steps:
+        - name: Checkout Actions
+          uses: actions/checkout@v4
+        - name: generate list of rule ids
+          id: rule_ids
+          shell: bash
+          run: |
+            export rule_ids=$(cat se_req_mve/models/experiment_flow/incose_rules.json | jq -r '. | map(.id) | join(",")' )
+            echo "RULE_IDS=$rule_ids" >> $GITHUB_OUTPUT
+
   execute-platform-pr-workflow:
     uses: ./.github/workflows/platform_pr_dev_workflow.yml
+    needs: create-rule-id-list
     with:
       env_name: ${{ inputs.env_name || 'pr'}}
       flow_type: ${{ inputs.flow_type || 'se_req_mve' }}
+      rule_ids: "${{ needs.create-rule-id-list.outputs.RULE_IDS }}"
```

Comment:
I think for a PR I might change this to just run 1 rule so it doesn't take as long. And leave all the rules for a merge

Reply:
what do you think is the overarching goal in this stage of the LLMOps? 

Use that to influence your decision.

![image](https://github.com/ISE-Crew-Poseidon/llmops-promptflow-template/assets/142918561/45bdc742-97a6-4eaa-95cd-0bacb4985601)

Reply:
I think the goal is to validate that the code is still working and can do a test run. I think running on all 8 rules is overkill for that and will also slow down the PR process because it will kick off all runs for every hypothesis for each commit. So I think all hypothesis for 1 rule should be sufficient

Reply:
i concur!

Reply:
Fixed